### DISCUSSION TITLE
avviare automazione ..

### DISCUSSION THREAD

**Discussion by RobyBS** [indomus/forum#1639] [HUB: Home Assistant]
Vi chiedo una cortesia .. 
il mio interruttore generale del contatore elettrica , avvia anche 3 shelly4pro ... 
cosa succede .. quando va via la corrente o banalmente "scatta" il magnetotermico, la si riattiva tramite l'interruttore generale posizionandolo da 0 a 1 ... 
lo shelly4pro si avvia ... ma tutti gli switch sono OFF e devo manualmente attivarli tramite i tastierini fisici ... 

pensavo di fare una automazione che , una volta tornata la corrente (attivo l'interruttore generale) dopo TOT secondi mi porti i vari switch impostati su shelly4pro da OFF a ON .. 
quando riattivo l'interruttore generale , ovviamente si avvia anche il modem ed il raspberry .. questi impiegano TOT secondi prima di essere operativi ... 

da qui ... l'automazione che voglio fare deve partire soltanto una volta , all'avvio del raspberry e di HA ovviamente (HASSIO), dopo i famigerati TOT secondi... 

come posso dare questo input in modo "elegante" all'automazione ?

**Comment by 50494554524F** [comment]
trigger con "platform homeassistant" ed evento "start", usa "for" per aspettare prima di eseguire lo script...

**Reply by Scialla** [reply]
cmq gli shelly hanno l'impostazione da settare in caso di riavvio; in pratica già nelle impostazioni dello shelly gli puoi dire di riaccendersi una volta tornata la corrente

**Reply by RobyBS** [reply]
grazie ad entrambi per le info ... vedrò come meglio fare per le mie necessità ... ;)
